docs/about/concepts/text/data-acquisition-concepts.md

```diff
@@ -12,7 +12,7 @@ modality: "text-only"
 
 # Data Acquisition Concepts
 
-This guide covers the core concepts for acquiring and processing text data from remote sources in NeMo Curator. Data acquisition focuses on downloading, extracting, and converting remote data sources into {ref}`DocumentDataset <documentdataset>` format for further processing.
```

Comment:
The `documentdataset` reference doesn't exist. I couldn't figure out what this was supposed to link to, so I just removed the link altogether. Happy to link to the right place if someone can point my in the right direction

Reply:
This ended up getting updated in another PR. Just merged `main` to resolve the merge conflict.